src/org/omegat/core/threads/SaveThread.java

```diff
@@ -89,16 +90,16 @@ public void run() {
         try {
             while (true) {
                 synchronized (this) {
-                    // Set flag for saving. If somebody will reset time, he will
-                    // clear this flag also.
+                    // Set the flag for saving. If somebody resets the timer,
+                    // one should clear the flag.
                     needToSaveNow = true;
```

Comment:
I don't want to get overly picky about the wording of comments, but:

`Set the flag for saving. Clear the flag if the timer is reset.`

would probably be clearer the next time a developer needs to look at this part of the code.